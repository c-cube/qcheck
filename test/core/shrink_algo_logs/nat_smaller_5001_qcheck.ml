open QCheck

let nats_smaller_5001 =
  Test.make ~name:"nat < 5001"
    (make ~shrink:Shrink.int Gen.nat) (Log.shrinks_int @@ fun n -> n < 5001)

let rand = Random.State.make [|1234|];;
try Test.check_exn ~rand nats_smaller_5001 with Test.Test_fail _ -> ()
