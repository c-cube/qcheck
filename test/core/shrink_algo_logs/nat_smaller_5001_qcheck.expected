holds 250
holds 7
holds 97
holds 174
holds 9
holds 2
holds 1
holds 85
holds 544
holds 57
holds 9
holds 68
holds 560
holds 0
holds 4
holds 44
holds 8
holds 206
holds 4
holds 590
holds 4
holds 75
holds 8
holds 8
holds 825
holds 4
fails 5624
holds 2812
holds 4218
holds 4921
fails 5273
holds 2637
holds 3955
holds 4614
holds 4944
fails 5109
holds 2555
holds 3832
holds 4471
holds 4790
holds 4950
fails 5030
holds 2515
holds 3773
holds 4402
holds 4716
holds 4873
holds 4952
holds 4991
fails 5011
holds 2506
holds 3759
holds 4385
holds 4698
holds 4855
holds 4933
holds 4972
holds 4992
fails 5002
holds 2501
holds 3752
holds 4377
holds 4690
holds 4846
holds 4924
holds 4963
holds 4983
holds 4993
holds 4998
holds 5000
fails 5001
holds 2501
holds 3751
holds 4376
holds 4689
holds 4845
holds 4923
holds 4962
holds 4982
holds 4992
holds 4997
holds 4999
holds 5000
